### ISSUE TITLE
pandas/numpy isn't working as expected

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#827]
Per @wildintellect:

>pandas/numpy in all the images seems broken - it's picking up an old Numpy version for some reason (path issue)

> just try to import pandas

**Comment by wildintellect** [comment]
conda list will show numpy 1.26.x but the error indicates 1.22.x is on the path

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
File /opt/conda/lib/python3.10/site-packages/pandas/__init__.py:23, in <module>
     21 try:
     22     # numpy compat
---> 23     from pandas.compat import (
     24         is_numpy_dev as _is_numpy_dev,  # pyright: ignore[reportUnusedImport] # noqa: F401,E501
     25     )
     26 except ImportError as _err:  # pragma: no cover

File /opt/conda/lib/python3.10/site-packages/pandas/compat/__init__.py:26, in <module>
     25 import pandas.compat.compressors
---> 26 from pandas.compat.numpy import is_numpy_dev
     27 from pandas.compat.pyarrow import (
     28     pa_version_under7p0,
     29     pa_version_under8p0,
   (...)
     33     pa_version_under14p0,
     34 )

File /opt/conda/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py:17, in <module>
     16 if _nlv < Version(_min_numpy_ver):
---> 17     raise ImportError(
     18         f"this version of pandas is incompatible with numpy < {_min_numpy_ver}\n"
     19         f"your numpy version is {_np_version}.\n"
     20         f"Please upgrade numpy to >= {_min_numpy_ver} to use this pandas version"
     21     )
     24 __all__ = [
     25     "np",
     26     "_np_version",
     27     "is_numpy_dev",
     28 ]

ImportError: this version of pandas is incompatible with numpy < 1.22.4
your numpy version is 1.22.3.
Please upgrade numpy to >= 1.22.4 to use this pandas version

The above exception was the direct cause of the following exception:

ImportError                               Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import pandas as pd

File /opt/conda/lib/python3.10/site-packages/pandas/__init__.py:28, in <module>
     26 except ImportError as _err:  # pragma: no cover
     27     _module = _err.name
---> 28     raise ImportError(
     29         f"C extension: {_module} not built. If you want to import "
     30         "pandas from the source directory, you may need to run "
     31         "'python setup.py build_ext' to build the C extensions first."
     32     ) from _err
     34 from pandas._config import (
     35     get_option,
     36     set_option,
   (...)
     40     options,
     41 )
     43 # let init-time option registration happen

ImportError: C extension: None not built. If you want to import pandas from the source directory, you may need to run 'python setup.py build_ext' to build the C extensions first.
```

**Comment by wildintellect** [comment]
Hmm, it seems to be working now? but only in some notebooks???

**Comment by wildintellect** [comment]
This should be solved by 3.1.4 release. Works fine in Vanilla.
